core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -313,7 +297,6 @@ private synchronized void closeContext() throws IOException {
       // developers should first mark their resources as closed prior to any exceptions being
       // thrown.
       mClosed.set(true);
-      MetricsSystem.removeMetrics(TOTAL_RPC_CLIENTS_METRICS_NAME);
```

Comment:
Removing is a good practice. But for some long running clients like HMS, I'm not sure if the FileSystemContext will be closed at all.